src/eddington_gui/app.py

```diff
@@ -345,6 +346,24 @@ def __calculate_fit_result(self):
                 title="Fit result error", message=str(error),
             )
             raise error
+   
+    def choose_sheet(self):
+        """Automatically choose the first valid sheet"""
+        for sheet in self.input_file_box.sheets_options:
+            try:
+                self.data_columns_box.fit_data = FitData.read_from_excel(
+                    Path(self.input_file_box.file_path),
+                    sheet)
+                self.input_file_box.children[1].children[1].value = sheet
+                return
+            except Exception as error:
+                pass
+        if self.data_columns_box.fit_data == None:
+            self.main_window.error_dialog(
+                title="Input data error",
+                message=str("No sheet available with valid data"))
```

Comment:
Two comments:
1. There is no need to wrap the message with `str()`
2. I would add to the message something like "please fix the file or load another one".